osquery/tables/events/linux/seccomp_events.cpp

```diff
@@ -0,0 +1,523 @@
+        {249, "request_key"},
+        {250, "keyctl"},
+        {251, "ioprio_set"},
+        {252, "ioprio_get"},
+        {253, "inotify_init"},
+        {254, "inotify_add_watch"},
+        {255, "inotify_rm_watch"},
+        {256, "migrate_pages"},
+        {257, "openat"},
+        {258, "mkdirat"},
+        {259, "mknodat"},
+        {260, "fchownat"},
+        {261, "futimesat"},
+        {262, "newfstatat"},
+        {263, "unlinkat"},
+        {264, "renameat"},
+        {265, "linkat"},
+        {266, "symlinkat"},
+        {267, "readlinkat"},
+        {268, "fchmodat"},
+        {269, "faccessat"},
+        {270, "pselect6"},
+        {271, "ppoll"},
+        {272, "unshare"},
+        {273, "set_robust_list"},
+        {274, "get_robust_list"},
+        {275, "splice"},
+        {276, "tee"},
+        {277, "sync_file_range"},
+        {278, "vmsplice"},
+        {279, "move_pages"},
+        {280, "utimensat"},
+        {281, "epoll_pwait"},
+        {282, "signalfd"},
+        {283, "timerfd_create"},
+        {284, "eventfd"},
+        {285, "fallocate"},
+        {286, "timerfd_settime"},
+        {287, "timerfd_gettime"},
+        {288, "accept4"},
+        {289, "signalfd4"},
+        {290, "eventfd2"},
+        {291, "epoll_create1"},
+        {292, "dup3"},
+        {293, "pipe2"},
+        {294, "inotify_init1"},
+        {295, "preadv"},
+        {296, "pwritev"},
+        {297, "rt_tgsigqueueinfo"},
+        {298, "perf_event_open"},
+        {299, "recvmmsg"},
+        {300, "fanotify_init"},
+        {301, "fanotify_mark"},
+        {302, "prlimit64"},
+        {303, "name_to_handle_at"},
+        {304, "open_by_handle_at"},
+        {305, "clock_adjtime"},
+        {306, "syncfs"},
+        {307, "sendmmsg"},
+        {308, "setns"},
+        {309, "getcpu"},
+        {310, "process_vm_readv"},
+        {311, "process_vm_writev"},
+        {312, "kcmp"},
+        {313, "finit_module"},
+        {314, "sched_setattr"},
+        {315, "sched_getattr"},
+        {316, "renameat2"},
+        {317, "seccomp"},
+        {318, "getrandom"},
+        {319, "memfd_create"},
+        {320, "kexec_file_load"},
+        {321, "bpf"},
+        {322, "execveat"},
+        {323, "userfaultfd"},
+        {324, "membarrier"},
+        {325, "mlock2"},
+        {326, "copy_file_range"},
+        {327, "preadv2"},
+        {328, "pwritev2"},
+        {329, "pkey_mprotect"},
+        {330, "pkey_alloc"},
+        {331, "pkey_free"},
+        {332, "statx"},
+        {333, "io_pgetevents"},
+        {334, "rseq"}};
+
+FLAG(bool,
+     audit_allow_seccomp_events,
+     false,
+     "Allow the audit publisher to process seccomp events");
+
+REGISTER(SeccompEventSubscriber, "event_subscriber", "seccomp_events");
+
+// namespace tables {
+// extern long getUptime();
+// }
+
+Status SeccompEventSubscriber::init() {
+  if (!FLAGS_audit_allow_seccomp_events) {
+    return Status(1, "Seccomp subscriber disabled via configuration");
```

Comment:
nitpick, the new convention is to use `Status::failure("Seccomp subscriber ...")`.